pkg/meta/redis.go

```diff
@@ -117,16 +117,18 @@ func newRedisMeta(driver, addr string, conf *Config) (Meta, error) {
 	var rdb *redis.Client
 	if strings.Contains(opt.Addr, ",") {
 		var fopt redis.FailoverOptions
-		ps := strings.Split(opt.Addr, ",")
+		ps := strings.Split(addr, ",")
```

Comment:
the `addr` may contain db (host:port/db), so we should use `opt.Addr` here

Reply:
Fixed